test/subscribe.spec.ts

```diff
@@ -331,4 +331,153 @@ describe('subscribe', () => {
 
   });
 
+  describe('ObservableData subscriber', () => {
+
+    class ModelA extends ObservableData {
+      foo: number;
+      b: any;
+    }
+
+    class ModelB extends ObservableData {
+      bar: string;
+      a: ModelA;
+    }
+
+    let target: ModelA;
+    let sub: SinonSpy;
+    let cancel: () => void;
+
+    beforeEach(() => {
+      target = new ModelA();
+      sub = spy();
+    });
+
+    describe('to primitive type', () => {
```

Comment:
verb missing, a little vague - maybe "(when) subscribe(d) to primitive type changes"?